Gems/Atom/RHI/Code/CMakeLists.txt

```diff
@@ -7,9 +7,26 @@
 #
 
 ly_get_list_relative_pal_filename(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME})
+ly_get_list_relative_pal_filename(pal_source_dir ${CMAKE_CURRENT_LIST_DIR}/Source/Platform/${PAL_PLATFORM_NAME})
 
 include(${pal_dir}/AtomRHITests_traits_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
 
+set(RENDERDOC_CMAKE ${CMAKE_CURRENT_SOURCE_DIR}/${pal_dir}/renderdoc_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
+if(EXISTS ${RENDERDOC_CMAKE})
+    include(${RENDERDOC_CMAKE})
+endif()
+
+if(TARGET "3rdParty::renderdoc")
```

Comment:
Where is 3rdParty::Renderdoc defined?  I searched globally for it, maybe my coodebase is older than this?

Also, doesn't this mean to integrate with renderdoc, you'd need to rebuild the engine to s upport it, ie, its not a runtime switch, its a define.  We're trying to avoid situations where someone has to recompile the engine to turn on a feature.  Is it not possible?  

It looks like from the PoV of the code, the vital piece is the render doc api initialization.  But REnderDoc itself appears to be MIT which would allow us to vendor or ship with those APIS at least available as header.  

This shouldn't necessarily block the distribution of this or committing ,but possible further work to be done to make it so that you can download the prebuilt installer and still "turn on renderdoc" without compiling...

Reply:
3rdParty is an namespace to the target added by the `ly_add_external_target` helper cmake function (see for example, https://github.com/o3de/o3de/pull/2680/files#diff-a146c60837341946a45da5cbda81df59e04082748d500eb6f505ccd0f0d7f228R20).

Do you mean that we could make it so the enduser can enable renderdoc and we would download/install it for them?